scripts/tests/py/metadata.py

```diff
@@ -0,0 +1,223 @@
+#!/usr/local/bin/python3
+# Copyright (c) 2022 Project CHIP Authors
```

Comment:
nit: (across all files) we could do copyright 2024. The others have 2023 which felt off, however 2022 is really old.